index.js

```diff
@@ -10,5 +11,18 @@ module.exports = {
       'apostrophe-seo-users'
     ],
     directory: 'lib/modules'
+  },
+  construct: function (self, options) {
+    self.prependSnippets = () => {
+      self.apos.templates.prepend('body', (req) => {
+        return self.partial('body-gtm', {});
```

Comment:
yaaasss

Reply:
is everything else also coming out this way or are we still making people include something in their layout at some point?

Reply:
We're still making people include it in their layout for BC. There could be a flag to do that, but it's not directly related to this work.